### DISCUSSION TITLE
I have problems modifying some canvass when using multiple instances of fabricjs

### DISCUSSION THREAD

**Discussion by w934423231** [fabricjs/fabric.js#8126] [Q&A]
My website has the function of tabs switching. There is a fabricjs instance in each tab page. When I have only one fabricjs instance, it is no problem to modify the properties of canvas, but when there are two instances, there is no rendering. I have seen the instance of fabricjs, and the parameters in it have been modified, and renderalll has also been executed

**Comment by w934423231** [comment]
![image](https://user-images.githubusercontent.com/38958128/182992654-61743eb7-e0fd-420d-85b6-d461c803404f.png)
![image](https://user-images.githubusercontent.com/38958128/182992716-9a7ed609-f70c-4aa0-a989-df7843aaa512.png)
![image](https://user-images.githubusercontent.com/38958128/182992784-d20380e0-a37b-4cee-8b7d-2171f7e6737b.png)

**Comment by ShaMan123** [comment]
version?
did you upgrade to latest?
sounds like you are reinitializing canvas which is an error and in the latest version throws

**Reply by w934423231** [reply]
I haven't updated the version since then, I'll try it tomorrow

**Reply by w934423231** [reply]
I use version 5.2.1
